basic/src/test/java/org/jboss/hal/testsuite/test/runtime/WebMetricsTestCase.java

```diff
@@ -93,6 +96,18 @@ public void before() throws IOException, TimeoutException, InterruptedException
         Console.withBrowser(browser).waitUntilLoaded();
     }
 
+    private void waitUntilRunning() throws TimeoutException {
+        String command = "/host=master/server=server-one:read-attribute(name=server-state)";
```

Comment:
you didn't define anywhere which are the servers, hardcoding isn't good thing to do. Having list of server for which to wait to start would make more sense.